pydantic/type_adapter.py

```diff
@@ -98,15 +115,33 @@ def _getattr_no_parents(obj: Any, attribute: str) -> Any:
         raise AttributeError(attribute)
 
 
+def _annotated_type(type_: Any) -> Any | None:
+    return get_args(type_)[0] if _typing_extra.is_annotated(type_) else None
+
+
```

Comment:
I'd assume we have this helper somewhere else that you could use, maybe I'm wrong